chapters/classes.tex

```diff
@@ -1544,6 +1544,8 @@ \subsection{Real Type}\label{real-type}
 \index{unbounded@\robustinline{unbounded}!attribute of \robustinline{Real}}%
 \index{stateSelect@\robustinline{stateSelect}!attribute of \robustinline{Real}}
 
+The following attributes shall be evaluable: \lstinline!quantity!, \lstinline!unit!, \lstinline!displayUnit!, \lstinline!fixed!, \lstinline!unbounded!, and \lstinline!stateSelect!.
```

Comment:
```suggestion
The following attributes shall be evaluable: \lstinline!quantity!, \lstinline!unit!, \lstinline!displayUnit!, \lstinline!fixed!, and \lstinline!stateSelect!.
```
Dymola 2023x Refresh 1 has support for not evaluating unbounded. Not only to be annoying, but since unbounded and nominal are treated together and evaluating min/max/nominal sometimes causes issues.

I don't know if we should say something for displayUnit as giving a value (or start-value) after translation with a different (display)Unit is fairly typical, even if I believe it _technically_ doesn't change the displayUnit for the variable (at least for Dymola - I don't know about other tools).

Reply:
OK, let's skip `unbounded` to keep it similar to `nominal`.